eng/pipelines/docindex.yml

```diff
@@ -56,127 +56,142 @@ jobs:
-          arguments: >-
-            -DocRepoLocation $(DocRepoLocation)
-            -OutputLocation $(DocRepoLocation)/docs-ref-mapping/reference-unified.yml
-        displayName: Generate ToC for Daily docs
-
-      - task: Powershell@2
-        inputs:
-          pwsh: true
-          filePath: eng/common/scripts/Verify-RequiredDocsJsonMembers.ps1
-          arguments: >-
-            -DocRepoLocation $(DocRepoLocation)
-        displayName: Verify Required Docs Json Members
-
-      - template: /eng/common/pipelines/templates/steps/git-push-changes.yml
-        parameters:
-          BaseRepoBranch: $(DailyDocsBranchName)
-          BaseRepoOwner: $(DocRepoOwner)
-          CommitMsg: "Update targeting packages based on release metadata. (Daily docs)"
-          TargetRepoName: $(DocRepoName)
-          TargetRepoOwner: $(DocRepoOwner)
-          WorkingDirectory: $(DocRepoLocation)
-          ScriptDirectory: $(Build.SourcesDirectory)/eng/common/scripts
-          PushArgs: -f
-
-      - task: AzureCLI@2
-        displayName: Queue Docs CI build
-        inputs:
-          azureSubscription: msdocs-apidrop-connection
-          scriptType: pscore
-          scriptLocation: inlineScript
-          inlineScript: |
-            $accessToken = az account get-access-token --resource "499b84ac-1321-427f-aa17-267ca6975798" --query "accessToken" --output tsv
-            $buildParamJson = '{"params":"{ \"target_repo\": { \"url\": \"https://github.com/MicrosoftDocs/azure-docs-sdk-node\", \"branch\": \"$(DailyDocsBranchName)\", \"folder\": \"./\" }, \"source_repos\": [] }"}'
-            eng/common/scripts/Queue-Pipeline.ps1 `
-              -Organization "apidrop" `
-              -Project "Content%20CI" `
-              -DefinitionId 3452 `
-              -BuildParametersJson $buildParamJson `
-              -BearerToken $accessToken
+      # Updating main is the default reason to kick off manual builds of the docIndex run.
+      # Manual builds of docIndex are typically done to update Main when certain updates are
+      # made. These updates include new libraries and CSV updates in azure-sdk which, depending
+      # on the column(s) updated can cause anything from package deprecation to service level
+      # readme or ToC updates.
+      - ${{ if or(eq(variables['Build.Reason'], 'Schedule'), ne(variables['Skip.MainUpdate'], 'true')) }}:
+        - task: Powershell@2
+          inputs:
+            pwsh: true
+            filePath: eng/common/scripts/Verify-RequiredDocsJsonMembers.ps1
+            arguments: >-
+              -DocRepoLocation $(DocRepoLocation)
+          displayName: Verify Required Docs Json Members
+
+        - task: Powershell@2
+          inputs:
+            pwsh: true
+            filePath: eng/common/scripts/Update-DocsMsPackageMonikers.ps1
+            arguments: -DocRepoLocation $(DocRepoLocation)
+          displayName: Move deprecated packages to legacy moniker
+          condition: succeeded()
```

Comment:
succeeded() by itself isn't needed and is redundant.

Reply:
They were all removed.